src/monitor-span-workspace/monitor-span.controller.ts

```diff
@@ -20,16 +20,17 @@ export class MonitorSpanWorkspaceController {
     return this.service.getSpans(locationId);
   }
 
-  @Get(':spanId')
+  @Put(':spanId')
   @ApiOkResponse({
     type: MonitorSpanDTO,
-    description: 'View a workspace span record for a monintor location',
+    description: 'Updates a workspace span record for a monintor location',
   })
   async getSpan(
```

Comment:
since this is a PUT the method name needs to be renamed appropriately to "updateSpan"